### ISSUE TITLE
Visual Studio Missing Java Binding Library Project Template

### ISSUE THREAD

**Issue by ColeXm** [xamarin/xamarin-android#5714]
In Visual Studio 16.8.5 everything works fine . 

But after I update Visual Studio 16.9.0, the template for android binding library is missing .

### Screen shot 

![Capture](https://user-images.githubusercontent.com/2161256/110443236-a98f8f00-80f6-11eb-9245-9652217fc706.PNG)

**Comment by AmrAlSayed0** [comment]
Yes, this is also the case in the latest Visual Studio 2019 v16.10 Preview 1

**Comment by joj** [comment]
This is fixed on preview 2. The workaround is to open an existing binding library instead of creating a new one. If anybody's interested I can create one and paste a zip here.

**Comment by AmrAlSayed0** [comment]
@mattleibow also face this issue https://discord.com/channels/732297728826277939/732297837953679412/814623538215518209

**Comment by xtuzy** [comment]
Visualstudio 16.9.2 , no Class Liberay(Android), no Binding Library. I'm so tired to xamarin for these bug.

**Comment by wangyankun33** [comment]
VS（windows）版本，v16.9.x 版本需要在设置中关闭某个功能，才能在新建项目中看到Xamarin.Android 绑定库，类库等选项

**Comment by jpobst** [comment]
Thanks @wangyankun33!

Unchecking the `Show all .NET Core templates in the New project dialog ` option will make the template show up.
![image](https://user-images.githubusercontent.com/179295/112490564-78959800-8d4d-11eb-9fcc-3e43861dfeda.png)

**Comment by EmilAlipiev** [comment]
Blank solution is also gone in the stable release. Crazy how VS team is doing updates

**Comment by sact1909** [comment]
> Thanks @wangyankun33!
> 
> Unchecking the `Show all .NET Core templates in the New project dialog ` option will make the template show up.
> ![image](https://user-images.githubusercontent.com/179295/112490564-78959800-8d4d-11eb-9fcc-3e43861dfeda.png)

Disable `Show all .NET Core templates in the New project dialog` worked for me, thanks

**Comment by maarkristof** [comment]
Templates are added back in the latest stable version (16.9.4).
